clang/test/CIR/CodeGen/inc-bool.c

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+
+#include <stdbool.h>
```

Comment:
We don't include headers like these as part of testing, you need to reduce the needed definitions and mock the content you need (see headers under `clang/test/CIR/Inputs` as examples).

Reply:
Fixed